english/mushroom/eat/eat_mushroom.md

```diff
@@ -1,4 +1,6 @@
-The mushroom makes great soup. You brought them back.
```

Comment:
This modification is fine, so if you respond to the other comments, I'll be happy to merge this.